Add a 2019 stats post with community updates (#1149)
writethedocs/www: docs/blog/write-the-docs-2019-stats.rst

.. post:: Jan 10, 2020
   :tags: stats, year in review, newsletter
   :author: Eric Holscher

Write the Docs 2019 Community Update
====================================

Since it's quiet over the holidays,
we normally don't publish a newsletter in January.
Instead,
we send out a yearly update for the Write the Docs community,
where we share updates on our progress as a community and an organization.

Community Updates
-----------------

There are a few things that we know some of you have been wondering about, and we'd like to provide updates on:

> The first big change is the `Write the Docs Enhancement Proposals (WEPs) <https://github.com/writethedocs/weps/>`_ process that we have implemented. This process aims to provide transparency in community-wide decision making, and allows the entire WTD community to review and vote on major policies, processes, and governance topics.

Reviewer: ```suggestion
The first big change is the `Write the Docs Enhancement Proposals (WEPs) <https://github.com/writethedocs/weps/>`_ process. This process aims to provide transparency in community-wide decision making, and allows the entire WTD community to review and vote on major policies, processes, and governance topics.
```

We already processed the first WEP, numbered WEP0_, 
which defines the system and how to use it.
We will be running more proposals through this process soon.
We hope that you will provide your feedback and insight to us as we look to guide the community into the future.

The second major thing is :doc:`another US conference </conf/cfp/east-2020>` closer to the east coast.

> We're still in the process of talking with venues and trying to see if will work for 2020.

Reviewer: ```suggestion
We're still in the process of talking with venues and trying to see if we can make it work for 2020.
```

> We will let everyone know the status of the event _before_ Portland tickets sell out,

Reviewer: ```suggestion
We will let everyone know the status of the event *before* Portland tickets sell out,
```

.. post:: Jan 10, 2020
   :tags: stats, year in review, newsletter
   :author: Eric Holscher

> Write the Docs 2019 Stats

Reviewer 2: ```suggestion
Write the Docs 2019 Community Update
```

=========================

Since it's quiet over the holidays,

> we normally don't do a newsletter in January.

Reviewer 2: ```suggestion
we normally don't publish a newsletter in January.
```

Instead,
we send out a yearly update for the Write the Docs community,

> filling you in on our progress.

Reviewer 2: ```suggestion
where we share updates on our progress as a community and an organization.
```

Community Updates
-----------------

> We have a few outstanding things that we wanted to update you on,

Reviewer: ```suggestion
There are a few things that we know some of you have been wondering about, and we'd like to provide updates on:
```

Deleting the next line also

coming into 2020.

> The first big change is the `Write the Docs Enhancement Proposals (WEPs) <https://github.com/writethedocs/weps/>`_ process that we have implemented.

Reviewer 2: ```suggestion
The first big change is the `Write the Docs Enhancement Proposals (WEPs) <https://github.com/writethedocs/weps/>`_ process that we have implemented. This process aims to provide transparency in community-wide decision making, and allows the entire WTD community to review and vote on major policies, processes, and governance topics.
```

> We have approved WEP0_,

Reviewer 2: ```suggestion
We already processed the first WEP, numbered WEP0_, 
```

> which defines the process,

Reviewer 2: ```suggestion
which defines the system and how to use it.
```

Reviewer: ```suggestion
which defines the enhancement proposal system and how to use it,
```

> and we will be running more proposals through this process soon.

Reviewer 2: ```suggestion
We will be running more proposals through this process soon.
```

We hope that you will provide your feedback and insight to us as we look to guide the community into the future.

The second major thing is :doc:`another US conference </conf/cfp/east-2020>` closer to the east coast.

> We've still in the process of talking with venues and trying to see if will work for 2020.

Reviewer 2: ```suggestion
We're still in the process of talking with venues and trying to see if will work for 2020.
```

We will let everyone know the status of the event _before_ Portland tickets sell out,
so that you won't miss your chance to attend.

The last major thing is our :doc:`Salary Survey </surveys/salary-survey-sep-2019>`,

> which finished up in November.

Reviewer 2: ```suggestion
which ran until November.
```

This is the first year that we've done this,

> and we were blown away by the response.

Reviewer 2: ```suggestion
and we were blown away by the response rate.
```

We had over 700 people fill it out,
and we're in the process of sifting through the data to provide some insights.
We plan to release the full results soon,

> and in the future hopefully we will have a smoother process for this in place to get it turned around faster.

Reviewer 2: ```suggestion
and in the future we hope to improve the process so that it gets analyzed and published faster.
```

.. note:: 
   

>    You can also read our posts from 2016_, 2017_, and 2018_.

Reviewer 2: I'd move this line to the bottom, or at least after the "stats" part, because it seems out of place after the community updates.

.. _WEP0: https://github.com/writethedocs/weps/blob/master/accepted/WEP0000.rst
.. _2018: https://www.writethedocs.org/blog/write-the-docs-2018-stats/
.. _2017: https://www.writethedocs.org/blog/write-the-docs-2017-stats/
.. _2016: https://www.writethedocs.org/blog/write-the-docs-2016-year-in-review/

> Conferences

Reviewer 2: I'd add a "2019 Statistics" heading here, and bump all the following headings one level down so that they're nested under that heading.

-----------

* 500 attendees in Portland (up from 450)
* 300 attendees in Prague (up from 275)
* 100 attendees in Australia (similar to last year)

We also had a conference in Vilnius this past year,
which drew 50 people.

Slack Network
-------------

* 8,250 :doc:`members </slack>` (up from 5,250)

Newsletter
----------

*  7,386 :doc:`subscribers </newsletter>` (up from 4,707)

Meetups
-------

> * TBD members (up from 8642)

Reviewer 2: Let's not forget this ;)

* 46 :doc:`meetups </meetups/index>` (up from 39)

Podcast
-------

> * 26 :doc:`episodes </podcast>` total (up from 18)

Reviewer 2: Is there a way to tracke listeners/subscribers?

Author: Not that I know of.

> Website

Reviewer 2: Do we have stats for Twitter? Maybe we can start tracking followers if it makes sense...

Reviewer: I'm less interestedin talking about twitter followers. Slack is about community, where people discuss things between themselves.  If really you want numbers here, tweets containing #writethedocs or something.

-------

* 250,000 sessions (same as last year)
* 430,000 page views (down from 450,000)

GitHub
------

> * 1,955 commits_ to our repository (up from 1,250)

Reviewer 2: Which repo is this? www?

Author: Yea, it's linked.

* 56 people_ who contributed to our repository (up from 53)

.. commits: git rev-list --count --all --after="2017-12-31" --before="2019-01-01"
.. _commits: https://github.com/writethedocs/www/commits/master
.. _people: https://github.com/writethedocs/www/graphs/contributors?from=2019-01-01&to=2020-01-01&type=c

Thanks
------

> Thanks again to everyone who makes this community possible. 

Reviewer 2: ```suggestion
Thanks again to everyone who makes this community possible. The community is the people, and we're grateful to share our passion for docs with so many delightful people all around the world. 

We hope to connect with you during 2020, whether in person at one of our conferences or online in one of our collaboration platforms. 

Happy new year!

The Write the Docs team
```
